app/controllers/patients_controller.rb

```diff
@@ -66,6 +68,18 @@ def destroy
     redirect_to patients_path
   end
 
+  def export
```

Comment:
I don't know if this is correct, but I remember Urbi telling me that in the controller, there should be nothing but CRUD functionality.
 
 @itsmeurbi help us here haha

Reply:
Not necessarily is wrong, but in terms of Rails it is better to tight to the CRUD actions
Always try to treat things as resources.

@Ballo1988 in this case, exporting patients is nothing more than requesting a list of patients in csv format; that is an index action

Same for the consultants. You're requesting a list of them in csv format

Please refer to https://apidock.com/rails/v7.1.3.2/ActionController/MimeResponds/respond_to